docs/user/04-metrics.md

```diff
@@ -347,6 +347,14 @@ spec:
 > [!NOTE]
 > The Metric agent can scrape endpoints even if the workload is a part of the Istio service mesh and accepts mTLS communication. However, there's a constraint: For scraping through HTTPS, Istio must configure the workload using 'STRICT' mTLS mode. Without 'STRICT' mTLS mode, you can set up scraping through HTTP by applying the annotation `prometheus.io/scheme=http`. For related troubleshooting, see [Log entry: Failed to scrape Prometheus endpoint](#log-entry-failed-to-scrape-prometheus-endpoint).
 
+
+### 4. Default Metrics for Telemetry Health
+By default, the metric gateway has the [kymastats receiver](https://github.com/kyma-project/opentelemetry-collector-components/blob/main/receiver/kymastatsreceiver/README.md) enabled. This receiver collects [metrics](https://github.com/kyma-project/opentelemetry-collector-components/blob/main/receiver/kymastatsreceiver/documentation.md) to monitor the health of following resources if they have been deployed:
```

Comment:
Please make it much more end-user oriented and do not talk about implementation details, like:

By default, a MetricPipeline will emit metrics about the health of all pipelines managed by the telemetry module. On base of this metrics, you can track the status of every individual pipeline and setup alerting for it. The following metric is emmitted for every pipeline and the telemetry module itself:
- xxx.conditions - the value represents the xxx

The following metric attributes are available on the metric:
- xxx - use it to narrow down the bla

A typical alert rule will look like that:

Reply:
;) And language-edited:

By default, a MetricPipeline emits metrics about the health of all pipelines managed by the Telemetry module. Based on these metrics, you can track the status of every individual pipeline and set up alerting for it. The following metric is [or, if there's more than one metric: "The following metrics are..."] emitted for every pipeline and the Telemetry module itself:

xxx.conditions - the value represents the xxx
The following metric attributes are available for the metric:

xxx - use it to narrow down the bla
A typical alert rule looks like the following example: